modules/internal/BytesStringCommon.chpl

```diff
@@ -183,6 +178,59 @@ module BytesStringCommon {
     return ret;
   }
 
+   /*
+   This function decodeHelp is used to create a wrapper for 
+   qio_decode_char_buf* and qio_decode_char_buf_esc and return 
+   the value of syserr , cp and nBytes.
+   :arg buff: Buffer to decode 
+   :arg buffLen: Size of buffer
+   :arg offset: Starting index of read buffer,
+   :arg allowEsc: Choice between "qio_decode_char_buf" 
+                  and "qio_decode_char_buf_esc" for escape statement
+   
+   :returns: Tuple of decodeRet, chr and nBytes
+   */
+
```

Comment:
Let's format the doc string better:
- no space between the proc and the string.
- indent the main paragraph once
- indent the arguments twice
- put space between the paragraph and args
- put space between args

To be clear, this doc will not show in our documentation because this is an internal helper, but it is good to keep things tidy.